src/sentry/digests/utilities.py

```diff
@@ -23,3 +24,96 @@ def get_digest_metadata(digest):
                     end = record.datetime
 
     return start, end, counts
+
+
+def get_personalized_digests(project_id, digest, user_ids):
+    """
+    get_personalized_digests(project: Project, digest: Digest, users: Set[User]) -> Iterator[User, Digest]
+    """
+    # TODO(LB): I Know this is inefficent.
```

Comment:
That's fine — it's probably worth clarifying here why though for future reference/review.